superset/charts/schemas.py

```diff
@@ -782,7 +783,7 @@ class ChartDataQueryObjectSchema(Schema):
         description="Reverse order. Default: `false`", required=False
     )
     extras = fields.Nested(ChartDataExtrasSchema, required=False)
-    columns = fields.List(fields.String(), description="",)
+    columns = fields.List(fields.String(), description="", allow_none=True)
```

Comment:
Same for `columns` as `groupby`.

Reply:
nit could be nice to add a description to `columns` and `extras`